home-tasks/src/test/java/com/epam/tc/hw1/functions/AdditionOperationTestDouble.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.tc.hw1.functions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.epam.tat.module4.Calculator;
+import com.epam.tc.hw1.providers.DataProviderForCalculator;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class AdditionOperationTestDouble {
+    Calculator calculator;
+
+    @BeforeMethod
```

Comment:
FYI, wisely choose method/class/... for configurational methods. Some actions should be performed for each test-method call, some, like initialization of SUT (system under test) should happen once per suite.